src/NuGet.Clients/NuGet.PackageManagement.UI/ViewModels/PackageItemViewModel.cs

```diff
@@ -136,6 +136,20 @@ public string ByOwnerOrAuthor
             }
         }
 
+        private IEnumerable<NuGetVersion> _installedVersions;
+        public IEnumerable<NuGetVersion> InstalledVersions
+        {
+            get
+            {
+                return _installedVersions;
+            }
+            set
+            {
+                _installedVersions = value;
+                OnPropertyChanged(nameof(InstalledVersions));
```

Comment:
This will be triggered when the InstalledVersions variable gets set to a new value, but not when a new element is appended to the list of Installed versions as on this line: https://github.com/NuGet/NuGet.Client/pull/5998/files?diff=unified&w=0#diff-07f06084bb40184699d53dcfdf13b4535adbb5b6ee122798b2a6b2c1d5bb9e31R274

If you want the View to know when a new InstalledVersion is appended to the list and refresh, you'll need to handle that case.

Reply:
Actually, the line that you mentioned will trigger this OnPropertyChanged because I'm doing an `=` operation.

Looking at the code we can do an improvement, changing this to an Observable list, this could save me doing the `=` when I'm already doing an append.